packages/devtools/lib/src/timeline/flame_chart.dart

```diff
@@ -14,6 +14,8 @@ import '../ui/flutter_html_shim.dart';
 import '../ui/theme.dart';
 import 'timeline.dart';
 
+// TODO(kenzie): delete this class once frame flame chart is ported to canvas.
```

Comment:
this comment is right in from of a const. Is the comment in the wrong place or is what you said to delete already deleted?